bin/gltf-pipeline.js

```diff
@@ -39,29 +41,35 @@ fs.readFile(gltfPath, function (err, data) {
     }
 
     var gltf = JSON.parse(data);
-    var stats = new OptimizationStatistics();
+    gltf = loadGltfUris(gltf, path.dirname(gltfPath), function(loadError) {
```

Comment:
I wasn't sure if there would be a problem since I have multiple `err` objects passed back in callbacks, but I will change it back!